metadata/metadata.ts

```diff
@@ -5252,4 +5252,14 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
       { trait_type: "Tradable", value: "No" },
     ],
   },
+  "Town Sign": {
+    description: "Show your farm ID with pride!",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/1284.png",
```

Comment:
There isn't a image in the files to show for OS metadata. Not a huge deal but do you need any help with the image?

Reply:
I created a quick image for the town sign. Maybe you could include it in.
![1284](https://github.com/sunflower-land/sunflower-land/assets/101262042/77a94128-7918-4e54-94e9-2cae74b04c6c)

Reply:
@elias7905 Good catch, forgot to generate this!!